test/storybook.test.js

```diff
@@ -173,12 +173,15 @@ describe('percy storybook', () => {
 
     await expectAsync(storybook(['http://localhost:8000']))
     // message contains the client stack trace
-      .toBeRejectedWithError(/^Story Error\n.*\/iframe\.html.*$/s);
+      .toBeRejectedWithError(/^Snapshot Name:/s);
```

Comment:
This is incorrect change

Reply:
fixed. Snapshot name is getting appended to the error.